### PR TITLE
Xis armorpack

### PR DESCRIPTION
**Pull request by Xisifer**
Contains:

- Helmet Pack 3
- Minor head 220 tweak
- Smuggler Jacket
- Mercenary Outfit (missing scarf textures, missing male bandolier model)
- Jedi Tunic

Note: ****Jedi and Mercenary pelvises are missing!! Models didn't compile, so I left them out of the PR****

### CODE REVIEW

File: swlor2_mdl/pfh0_pelvis009.mdl [hunk 1]
```diff
@@ -0,0 +1,5415 @@
+# Exported from NeverBlender 2.8.33 - Sunday, 2022-05-08
```
Comment (zunath) [inline swlor2_mdl/pfh0_pelvis009.mdl:1]: Need compilation on this file please

File: swlor2_mdl/pfh0_pelvis036.mdl [hunk 2]
```diff
@@ -0,0 +1,5015 @@
+# Exported from NeverBlender 2.8.33 - Sunday, 2022-05-08
+filedependancy ARMORPACK.blend
+newmodel pfh0_pelvis036
+setsupermodel pfh0_pelvis036 null
+classification CHARACTER
+setanimationscale 1.0
```
Comment (zunath) [inline swlor2_mdl/pfh0_pelvis036.mdl:6]: Compilation here too please

File: swlor2_mdl/pmh0_belt153.mdl [hunk 3]
```diff
@@ -0,0 +1,10718 @@
+# Exported from NeverBlender 2.8.33 - Sunday, 2022-05-08
+filedependancy ARMORPACK.blend
```
Comment (zunath) [inline swlor2_mdl/pmh0_belt153.mdl:2]: One here too

File: swlor2_mdl/pmh0_chest052.mdl [hunk 4]
```diff
@@ -0,0 +1,2174 @@
+# Exported from NeverBlender 2.8.33 - Saturday, 2022-05-07
```
Comment (zunath) [inline swlor2_mdl/pmh0_chest052.mdl:1]: And here

File: swlor2_mdl/pmh0_pelvis009.mdl [hunk 5]
```diff
@@ -0,0 +1,1656 @@
+# Exported from NeverBlender 2.8.33 - Sunday, 2022-05-08
+filedependancy ARMORPACK.blend
```
Comment (zunath) [inline swlor2_mdl/pmh0_pelvis009.mdl:2]: Compilation here please

File: swlor2_mdl/pmh0_pelvis157.mdl [hunk 6]
```diff
@@ -0,0 +1,4668 @@
+# Exported from NeverBlender 2.8.33 - Sunday, 2022-05-08
```
Comment (zunath) [inline swlor2_mdl/pmh0_pelvis157.mdl:1]: Compilation here please

File: swlor2_mdl/pmh0_belt153.mdl (hunk 3 above)
Comment (Xisifer) [reply swlor2_mdl/pmh0_belt153.mdl:2]: This model actually is crashing my game when trying to load it. On the rare occasion I CAN load it, it's got gunky stretched polygons. Gonna have to leave it out of the PR until I can figure out how to fix it.

File: swlor2_mdl/pfh0_pelvis036.mdl (hunk 2 above)
Comment (Xisifer) [reply swlor2_mdl/pfh0_pelvis036.mdl]: Compiled

**Comment by Xisifer** [comment]
Skinmeshed models have been compiled. 

pmh0_belt153.mdl is crashing my game, so I'm removing it from the PR until I can fix it.

**Review by zunath — APPROVED** [review]

**Pull request merged**